docs/astro/src/content/docs/guide/tooling/figma-inspector.mdx

```diff
@@ -0,0 +1,88 @@
+---
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+title: Figma Variable Export
+description: Exporting Figma Variables to Slint Design Tokens
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import LangRefLink from '/src/components/LangRefLink.astro';
+
+## Setting up the Figma Inspector
+First of all, find and enable the "Figma to Slint" plugin in the Figma Plugin Manager.
+The inspector currently has 2 functions.  One is to inspect individual Figma objects for their properties, and the other is to export the variables defined in Figma so they are useable in Slint directly.
+
+## Key Features of Variable Export
```

Comment:
This looks a little like a "marketing brochure" to me. I suggest to break this down into sections, perhaps with groups?

For example, how we map figma variable types to Slint types is IMO best represented using a table, instead of just one sentence.

We have a similar scenario with the Rust code that we generated from Slint files, and besides an overview explanation, we also provide an example of how the generated code looks like (in structure, not content). Perhaps the same could be used here. ("Suppose the following variables in figma <insert screenshot>, we generate the following Slint code")